src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -1444,3 +1484,102 @@ ActStatus ActionHandlers::handle_state_guard_timeouts_for_csreq_ind(ControlBlock
     mme_tables->initiate_spgw_resolution();
     return ActStatus::ABORT;
 }
+/***************************************
+* Action handler : req_to_dns
+***************************************/
+ActStatus ActionHandlers::req_to_dns(ControlBlock& cb)
+{
+	log_msg(LOG_DEBUG, "Inside req_to_dns \n");
+	 if(mme_cfg->dns_config.dns_flag == 1)
+        {
+
+	uint8_t plmn_id[3] = {0};
+
+        UEContext *ue_ctxt = dynamic_cast<UEContext*>(cb.getPermDataBlock());
+        if (ue_ctxt == NULL)
+        {
+                log_msg(LOG_DEBUG, "send_air_to_hss: ue context is NULL \n");
```

Comment:
update log.

Reply:
changes the log to log_msg(LOG_DEBUG, "req_to_dns: ue context is NULL \n");